frontend/src/components/App.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { Switch, Route, HashRouter } from 'react-router-dom';
-
 import './App.css';
+import './App.sass';
```

Comment:
I think we should try to keep the stylesheet files to one type so it won't be confusing trying to find or debug something.

Reply:
I can do that, I am not using that file at all, I just left it in case someone else does not know how to use sass and wanted to use css. I will take it out. thanks 👍